ansible/roles/proteus-events/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: create Proteus group
+  group: name={{ proteus_group }}  state=present
+
+- name: create Proteus user
+  user:
+    name: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    createhome: no
+    shell: /sbin/nologin
+    comment: "Proteus User"
+    state: present
```

Comment:
Minutes if everything is OK, couple of hours if something goes seriously wrong :)